server/auth_test.go

```diff
@@ -62,6 +66,11 @@ var _ = Describe("Auth", func() {
 		})
 
 		Describe("Login from HTTP headers", func() {
+			const TRUSTED_IPV4 = "192.168.0.42"
+			const UNTRUSTED_IPV4 = "8.8.8.8"
+			const TRUSTED_IPV6 = "2001:4860:4860:1234:5678:0000:4242:8888"
+			const UNTRUSTED_IPV6 = "5005:0:3003"
```

Comment:
Please use camelCase (Go convention)
```suggestion
			const (
			    trustedIPV4 = "192.168.0.42"
			    untrustedIPV4 = "8.8.8.8"
			    trustedIPV6 = "2001:4860:4860:1234:5678:0000:4242:8888"
			    untrustedIPV6 = "5005:0:3003"
			)
```